scripts/vaccination_alerts/parse-vaccination-phases.ts

```diff
@@ -0,0 +1,168 @@
+import fetch from 'node-fetch';
+import { groupBy, sortBy } from 'lodash';
+import * as yargs from 'yargs';
+import path from 'path';
+import fs from 'fs-extra';
+import {
+  stateVaccinationPhases,
+  RegionVaccinePhaseInfo,
+} from '../../src/cms-content/vaccines/phases';
+
+import regions from '../../src/common/regions/region_db';
+
+const parse = require('csv-parse/lib/sync');
+
+const PHASES_PATH = path.join(
+  __dirname,
+  '..',
+  '..',
+  'src',
+  'cms-content',
+  'vaccines',
+  'state-vaccine-phases.json',
+);
+
+const GOOGLE_SHEETS_CSV_URL =
+  'https://docs.google.com/spreadsheets/d/e/2PACX-1vQjhGx1L-_kWAwfOdeVbVEa9QSI5Uq9sL6xTqOk-vU0k2ncIDhRIJKBdURBlKebqrPcTey_iOw2224l/pub?gid=0&single=true&output=csv';
+
+interface CsvRow {
+  state: string;
+  eligibilityUrl: string;
+  currentlyEligible: boolean;
+  phase: string;
+  tier: string;
+  startDate: string;
+  groupDetails: string;
+  generalGroup: string;
+  descriptionUrl: string;
+  notes: string;
+}
+
+const parseRow = (row: { [key: string]: string }): CsvRow => {
+  return {
+    state: row['State'],
+    eligibilityUrl: row['Eligibility Information URL'],
+    currentlyEligible: row['Currently Eligible'].trimRight() !== '',
+    startDate: row['Estimated Start Date'],
+    phase: row['Phase'],
+    tier: row['Tier'],
+    groupDetails: row['Detailed Group'],
+    generalGroup: row['General Group'],
+    descriptionUrl: row['Expanded description url'],
+    notes: row['Notes for Internal QA'],
```

Comment:
Can we use indices instead to make this more resilient?

Reply:
hmm I'm not sure that will really make it more resilient - very possible that a column is added and breaks it.  with indices I'm more concerned the failure will be silent